lib/core/src/lib/form/components/widgets/core/form-field-validator.ts

```diff
@@ -583,3 +584,39 @@ export const FORM_FIELD_VALIDATORS = [
     new MinDateTimeFieldValidator(),
     new MaxDateTimeFieldValidator()
 ];
+
+export const momentToDateFnsMap = {
+    M: 'M',
+    D: 'd',
+    Y: 'y',
+    A: 'a'
+};
+
+export const dateFnsToMomentMap = {
+    M: 'M',
+    d: 'D',
+    y: 'Y',
+    a: 'A'
+};
+
+export function convertMomentToDateFnsFormat(dateDisplayFormat: string): string {
```

Comment:
provide documentation comments for public functions

Reply:
added documentation